enterprise/coderd/prebuilds/reconcile_test.go

```diff
@@ -0,0 +1,741 @@
+package prebuilds_test
```

Comment:
A concern that I have with these tests is we don't exercise the normal `RunLoop` which uses the defined `ReconciliationInterval`. I suspect other reviewers will raise this, so we should consider adding one. Along with that, adding quartz would be useful to ensure the interval is behaving as expected.

Reply:
@dannykopping I added `TestRunLoop`: https://github.com/coder/coder/pull/17261/commits/9ac7a2c7198874c3b3b67d10da8c1832ae3d20b4

wdyt is it enough?

Reply:
LGTM 👍 thanks!